docs/data-cookbooks/cnm-workflow.md

```diff
@@ -233,11 +235,13 @@ Below is a description of each of these tasks:
 
 You can also manipulate the data sent to downstream tasks using `task_config` for various states in your workflow resource configuration. Read more about how to configure data on the [Workflow Input & Output](https://nasa.github.io/cumulus/docs/workflows/input_output) page.
 
+Documentation for `CNMToCMA` Lambda package is provided in the [source repository](https://git.earthdata.nasa.gov/projects/POCUMULUS/repos/cnmtogranule/browse).
```

Comment:
I think we should write something like: This is currently only available for NASA Bitbucket users. Or something like that. Or maybe note at the top of this cookbook that you need certain access for this to be applicable.